c/BUILD.bazel

```
@@ -65,9 +65,73 @@ cc_test(
         "test/check_bitfield_macros.c",
         "test/check_suites_legacy.h",
     ] + SBP_LEGACY_C_SOURCES,
+    includes = [
+        "include/libsbp",
+    ],
+    deps = [
+        ":sbp",
+        "@my-check//:check",
+    ],
+)
+
+SBP_V4_C_SOURCES = glob(["test/auto*.c"])
+
+cc_test(
+    name = "sbp-v4-test",
+    srcs = [
+        "test/check_main.c",
+        "test/check_edc.c",
+        "test/check_sbp.c",
+        "test/check_bitfield_macros.c",
+        "test/check_suites.h",
+    ] + SBP_V4_C_SOURCES,
     includes = ["include/libsbp"],
     deps = [
         ":sbp",
         "@my-check//:check",
     ],
 )
+
+SBP_CPP_V4_C_SOURCES = glob(["test/cpp/auto*.cc"])
+
+cc_test(
+    name = "sbp-cpp-v4-test",
+    srcs = SBP_CPP_V4_C_SOURCES,
+    deps = [
+        ":sbp",
+        "@my-googletest//:gtest_main",
+    ],
+)
+
+cc_test(
+    name = "sbp-string-test",
+    srcs = [
+        "test/string/test_double_null_terminated.cc",
+        "test/string/test_multipart.cc",
+        "test/string/test_null_terminated.cc",
+        "test/string/test_unterminated.cc",
+    ],
+    includes = [
+        "src/include",
+    ],
+    deps = [
+        ":sbp",
+        "@my-googletest//:gtest_main",
+    ],
+)
+
+SBP_CPP_LEGACY_SOURCES = glob(["test/legacy/cpp/auto*.cc"])
+
+cc_test(
+    name = "sbp-cpp-legacy-test",
+    srcs = [
+        "test/legacy/cpp/test_sbp_stdio.cc",
+    ] + SBP_CPP_LEGACY_SOURCES,
+    data = [
+        "test/legacy/cpp/sbp_data/gnss_data.sbp",
+    ],
+    deps = [
+        ":sbp",
+        "@my-googletest//:gtest_main",
+    ],
```

```suggestion
    ],
    tags = ["exclusive"],
```
This attribute will make it so that the test run serially after the other tests. I don't think it's as granular as making the individual tests themselves run in serial but it might be worth a shot.

---

Doc here: https://bazel.build/reference/be/common-definitions#common-attributes-tests

---

Today, even when I run all of the tests at once, it works for me with gcc-11.
Clang-14 still doesn't work, even with the "exclusive" tag.
The behavior is so unpredictable. I will create a ticket about it.